.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "42crunch.vscode-openapi"
+    ]
+}
```

Comment:
Since we are improving Dev Exp in this PR, why not suggest OpenAPI extension in vscode? Its a nice extension